pkg/controller/persistentvolume/persistentvolume_recycler_controller.go

```diff
@@ -35,6 +35,8 @@ import (
 	"k8s.io/kubernetes/pkg/watch"
 )
 
+var _ volume.VolumeHost = &PersistentVolumeRecycler{}
```

Comment:
This is an interface assertion.   It verifies that PersistentVolumeRecycler implements VolumeHost.